AirSkyBoat/AirSkyBoat
# Fix several dyna and hnm issues

**_I affirm:_**


## Please enter a player-facing description
- A rouge Nightmare Sabotender will no longer spawn a Nightmare Manticore in Dynamis-Valkurm. (Tracent)
- Quadavs in Dynamis-Valkurm will now drop the correct items. (Tracent)
- Vrtra pets will now share hate with Vrtra. (Tracent)

## What does this pull request do? (Please be technical)
This PR fixes two issues with the main issue that quadavs in Dynamis-Valkurm were dropping loot from the table of Dyna-Qufim NMs. Also fixes an issue where Vrtra pets had the incorrect targid and thus were not sharing hate with Vrtra.

## Steps to test these changes
Fight the quadavs in dyna valk and make sure they only drop the following loot:
WAR Feet, MNK Feet, WHM Feet, BLM Feet, RDM Feet, THF Head, PLD Feet, DRK Feet, BST Head, BRD Feet, RNG Hands, SAM Feet, NIN Legs, DRG Legs, SMN Legs and Byne Bill
 
Fixes some of the issues in https://github.com/HorizonFFXI/HorizonXI-Issues/issues/1579
## Special Deployment Considerations
